adapters/rubicon/rubicon_test.go

```diff
@@ -698,15 +698,44 @@ func TestDifferentRequest(t *testing.T) {
 	// test invalid size
 	pbReq.Bidders[0].AdUnits[0].Sizes = []openrtb.Format{
 		{
-			W: 0,
-			H: 0,
+			W: 2222,
+			H: 333,
 		},
 	}
+	pbReq.Bidders[0].AdUnits[1].Sizes = []openrtb.Format{
+		{
+			W: 222,
+			H: 3333,
+		},
+		{
+			W: 350,
+			H: 270,
+		},
+	}
+	pbReq.Bidders[0].AdUnits = pbReq.Bidders[0].AdUnits[:len(pbReq.Bidders[0].AdUnits)-1]
 	_, err = an.Call(ctx, pbReq, pbReq.Bidders[0])
 	if err == nil {
-		t.Fatalf("Should have gotten an error: %v", err)
+		t.Fatalf("Should have reported an error")
 	}
 
```

Comment:
Sorry I made some bad examples in this unit test, I'll fix them soon. You may want to check the length of the bids result in addition to checking whether err is nil. You can replace your code line 716 with following code:

        var bids pbs.PBSBidSlice
	bids, err = an.Call(ctx, pbReq, pbReq.Bidders[0])
	if len(bids) != 0 {
		t.Fatalf("Should have length as 0 instead of %v", len(bids))
	}

Reply:
yeah got a bit lazy there but updated them! Thanks! 👍